src/screens/Favorites/Favorites.styles.js

```diff
@@ -0,0 +1,62 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  containerMovies: {
+    marginTop: 50,
+  },
+
+  movieListContainer: {
+    flex: 1,
+    backgroundColor: '#000000',
```

Comment:
Colors should not be in the `styles.js` file because it can jeopardize theming and lead to large refactorings if someday we decide to add multiple styles (such as light/dark mode). 

All colors should be referenced from `theme/theme.js` and not from style variables (unless the color stays the same regardless of the theme)

Reply:
Done!